syncer/syncer.go

```diff
@@ -2894,7 +2895,6 @@ func (s *Syncer) printStatus(ctx context.Context) {
 				}
 			}
 
-			ctx2, cancel2 := context.WithTimeout(ctx, utils.DefaultDBTimeout)
 			latestMasterPos, latestmasterGTIDSet, err = s.getMasterStatus(ctx2)
```

Comment:
Maybe use another context ctx3?

Reply:
```suggestion
			ctx2, cancel2 = context.WithTimeout(ctx, utils.DefaultDBTimeout)
```

Reply:
call previous cancel2 before it's overwritten

Reply:
fixed, PTAL